internal/namespaces/k8s/v1beta4/custom_kubeconfig_install_test.go

```diff
@@ -0,0 +1,150 @@
+package k8s
+
+import (
+	"io/ioutil"
+	"os"
+	"path"
+	"testing"
+
+	"github.com/alecthomas/assert"
+	"github.com/scaleway/scaleway-cli/internal/core"
+	k8s "github.com/scaleway/scaleway-sdk-go/api/k8s/v1beta4"
+	"gopkg.in/yaml.v2"
+)
+
+var (
+	existingKubeconfigs = `apiVersion: v1
```

Comment:
can we indent the multiline string with the rest of the code ? 

we could use a function that removes tabs at the beginning, and empty lines

Reply:
I'm really not fond of this idea. It brings complexity for nothing. We could move the variable in its own file though